mkosi/__init__.py

```diff
@@ -5527,7 +5527,7 @@ def find_skeleton(args: argparse.Namespace) -> None:
 def args_find_path(args: argparse.Namespace, name: str, path: str, *, as_list: bool = False) -> None:
     if getattr(args, name) is not None:
         return
-    path = Path(path).absolute()
+    path: Path = Path(path).absolute()
```

Comment:
Ditto

Reply:
Renamed it `abspath`, for now. I think the proper solution would be to do what you started in #790 and add constants like `MKOSI_EXTRA = Path('mkosi.extra')` and so on for all special files, then this function could easily just take a `Path`, too.